js/build.js

```diff
@@ -16,7 +16,7 @@
   }
 
   Fliplet.Widget.instance('video-offline', function (data) {
-    var url = _.get(data, 'file.video.bundledFile.url');
+    var url = _.get(data, 'file.video.bundledFile.url') || _.get(data, 'file.video.selectFiles.url');
```

Comment:
Why is this change necessary?

Reply:
Because if `bundledFile.url` isn't available, for old widget instances, then the video source will be based on the `selectFiles.url` property. That would need to be checked and authenticated if necessary.